backends/qualcomm/runtime/backends/htpbackend/HtpGraph.h

```diff
@@ -23,10 +23,9 @@ class HtpGraph : public QnnGraph {
       QnnBackend* backend,
       QnnContext* context,
       const QnnExecuTorchProfileLevel& profile_level,
-      const std::string& graph_name,
```

Comment:
interesting - what did we use this `graph_name` before?

Reply:
Before we use it to store `graph_name` from compiler specs in AoT or from context binary in runtime.